### PR TITLE
testing/php7-pecl-seaslog: new aport

### PR DESCRIPTION
**Pull request by viest**
http://pecl.php.net/package/SeasLog
An effective,fast,stable log extension for PHP.

### CODE REVIEW

**Comment by wujunze** [comment]
LGTM

File: testing/php7-pecl-seaslog/APKBUILD [hunk 1]
```diff
@@ -0,0 +1,34 @@
+# Contributor: viest <dev@service.viest.me>
+# Maintainer: Chitao.Gao <neeke@php.net>
+pkgname="php7-pecl-seaslog"
+_pkgreal=SeasLog
+pkgver="2.0.2"
+pkgrel=0
+pkgdesc="An effective,fast,stable log extension for PHP."
+url="https://pecl.php.net/package/SeasLog"
+arch="all"
+license="PHP 3.01"
```
Comment (andypost) [inline testing/php7-pecl-seaslog/APKBUILD]: It should be `PHP-3.01` according to https://spdx.org/licenses/PHP-3.01.html

File: testing/php7-pecl-seaslog/APKBUILD [hunk 2]
```diff
@@ -0,0 +1,34 @@
+# Contributor: viest <dev@service.viest.me>
+# Maintainer: Chitao.Gao <neeke@php.net>
+pkgname="php7-pecl-seaslog"
+_pkgreal=SeasLog
+pkgver="2.0.2"
+pkgrel=0
+pkgdesc="An effective,fast,stable log extension for PHP."
```
Comment (andypost) [inline testing/php7-pecl-seaslog/APKBUILD]: nitpick but better to add spaces after `,`

File: testing/php7-pecl-seaslog/APKBUILD [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+    make NO_INTERACTION=1 REPORT_EXIT_STATUS=1 test
+}
+
+package() {
+    make INSTALL_ROOT="$pkgdir/" install
+    install -d "$pkgdir"/etc/php7/conf.d
+    echo "extension=$_pkgreal.so" > "$pkgdir"/etc/php7/conf.d/$_pkgreal.ini
+}
+
+sha512sums="bc1dc5855461c510edfe2a4cb50cee18b98b8063bbd07957300842073530cd7deae54e3c062ca175e11df9108dde4341cc75ba78376a3b95516509a9c2a0beae  php7-pecl-seaslog-2.0.2.tar.gz"
```
Comment (andypost) [inline testing/php7-pecl-seaslog/APKBUILD]: nit: no new line, use `abuild checksum` to fix or just manually

**Comment by viest** [comment]
@andypost   Thank you for your certification review, which makes my format more standard.

File: testing/php7-pecl-seaslog/APKBUILD (hunk 3 above)
Comment (Cogitri) [reply testing/php7-pecl-seaslog/APKBUILD]: I think the newline is still missing, makes for kind of ugly diffs :/

**Pull request closed without merging**